prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet

```diff
@@ -0,0 +1,24 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'prow-stale',
+        rules: [
+          {
+            alert: 'Prow images are stale',
+            expr: |||
+              time()-max(prow_version) > %d * 24 * 3600
+            ||| % $._config.prowImageStaleByDays,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: '@test-infra-oncall The prow images are older than %d days.' % $._config.prowImageStaleByDays,
```

Comment:
Please remove the `@test-infra-oncall`. That is specific to k8s-prow not istio, is not specific to the alert, and only makes sense for a slack alerting sinks. We should include this kind of thing in the alert routing config (but probably not at all here since we don't have an oncall slack team for istio AFAIK).

Reply:
done